tests/e2e.test.js

```diff
@@ -180,11 +169,6 @@ describe('Search', () => {
     expect(await isHidden($searchClear)).toBeTruthy();
   });
 
-  it('does not show the "no results" message on load', async () => {
```

Comment:
Would you mind to update these tests rather than removing them?

Reply:
Updated.